include/tvm/relay/feature.h

```diff
@@ -50,11 +50,18 @@ enum Feature : int {
   fMatch = 14,
   /*! \brief Whether any non-atom fragment of the program is shared, making the program a graph. */
   fGraph = 15,
+  /*!
+   * \brief Whether any complex(non primitive) fragment of the program is shared,
+   * making the program a graph.
+   * This is different from fGraph in the sense that if graph is used only in primitive function,
+   * fCGraph is off while fGraph is on.
+   */
+  fCGraph = 16,
```

Comment:
Nitpick: is this the best name? It's very similar to `fGraph` above, even though they mean different things

Reply:
I have no idea what name I should give.

Reply:
I don't know the best name myself. Perhaps rename `fGraph` to `fAtomGraph` and `fCGraph` to `fComplexGraph`? A little verbose but might be worth it to avoid mistakes

Reply:
fGraph dont really mean fAtomicGraph.

Reply:
What is the difference then? Is there any way to have a more specific name?

Reply:
fCGraph is graph outside of primitive function (but a primitive funciton might still has complex construct). I dont really know a better name.